python/mxnet/contrib/onnx/mx2onnx/_op_translations.py

```diff
@@ -2047,3 +2047,29 @@ def convert_broadcast_to(node, **kwargs):
     )
 
     return [tensor_node, expand_node]
+
+@mx_op.register("UpSampling")
+def convert_upsample(node, **kwargs):
+    """Map MXNet's UpSampling operator attributes to onnx's Upsample operator
+    and return the created node.
+    """
+    name, input_nodes, attrs = get_inputs(node, kwargs)
+
+    sample_type = attrs.get('sample_type', 'nearest')
+    sample_type = 'linear' if sample_type == 'bilinear' else sample_type
+    scale = convert_string_to_list(attrs.get('scale'))
+    scaleh = scalew = float(scale[0])
+    if len(scale) > 1:
+        scaleh = float(scale[0])
+        scalew = float(scale[1])
```

Comment:
is scale_h at index 0 or scale_w?

Reply:
should be index 0 i believe--ill change the code to be more clear.

Reply:
If an int scale is provided, both scaleh and scalew are given that value. However, if a tuple of scales is given, scaleh is assigned the first value and scalew is assigned the second value.